I.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,9 @@
+function paintShop(arrayOfCars, suggestedColor) {
+  // I used the line below to make a copy without refrence of the array of object.
+  var newArr = Array.from(JSON.parse(JSON.stringify(arrayOfCars)));
```

Comment:
I am not sure this is the best solution but it's the best I found.

Reply:
Seems like a reasonable approach